src/reducers/columnStateHelper.js

```diff
@@ -29,15 +30,9 @@ const DRAG_SCROLL_BUFFER = 100;
  * @return {!Object}
  */
 function initialize(state, props, oldProps) {
-  const { scrollLeft, scrollToColumn } = props;
   let { columnResizingData, isColumnResizing, scrollX } = state;
 
-  if (scrollLeft !== undefined &&
-      (!oldProps || scrollLeft !== oldProps.scrollLeft)) {
-    scrollX = scrollLeft;
-  }
-
-  scrollX = scrollTo(state, props, oldProps.scrollToColumn, scrollX);
+  const columnAnchor = getColumnAnchor(state, props, oldProps);
```

Comment:
Refactored into new function

* because column anchor can also used in the reducer for scrollX changes
* consistency with scrollY calculation (for rows)